### PR TITLE
Adding Additional Chore Phrases

### PR DESCRIPTION
**Pull request by bryan-johns**

### CODE REVIEW

File: utilities/randomPhrases.js [hunk 1]
```diff
@@ -6,20 +6,30 @@ const phrases = {
     ":wave: What's up, crew?",
     "It's that time of evening again!",
     ":wave: Whatup peeps?",
-    "Drumroll please...:drum_with_drumsticks:"
+    "Drumroll please...:drum_with_drumsticks:",
+    "Hey there ambulance crew!:ambulance:",
+    "Hey there crew!:grin:",
+    "Howdy there!:face_with_cowboy_hat:",
+    "Hey crew!:v:"
```
Comment (ddbruce) [inline utilities/randomPhrases.js]: Please add a space between the text and the emoji

**Comment by bryan-johns** [comment]
@ddbruce, changes added

File: utilities/randomPhrases.js [hunk 2]
```diff
@@ -7,10 +7,10 @@ const phrases = {
     "It's that time of evening again!",
     ":wave: Whatup peeps?",
     "Drumroll please...:drum_with_drumsticks:",
```
Comment (justetz) [inline utilities/randomPhrases.js]: This one also needs a space before the emoji, sorry @bryan-johns

Comment (ddbruce) [reply utilities/randomPhrases.js]: uff da

**Pull request merged**